go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Nextdoor/conductor
 
-go 1.12
+go 1.11.13
```

Comment:
Why not 1.12?  Is there an incompatibility?

Reply:
yes, I was making it consistent with what I had set in the Dockerfile, so local and docker have same version. Also 1.11.13 seems to work better within docker while downloading modules.